### PR TITLE
Fixes #80: Increases the contrast of text / background color in the demos.

### LINKED ISSUE #80: Mobile A11y Testing 11/1 - contrast check
This is largely fine, but please check the contrast level. It is very difficult for me to read the text of the non-selected items on top of that blue background, especially on a small mobile device.

### PR DESCRIPTION
**Pull request by bicknellr**
before | after
------ | -----
screen shot 2015-12-02 at 14 57 29 | screen shot 2015-12-02 at 14 57 43

### CODE REVIEW

**Comment by cdata** [comment]
@bicknellr could you include some before and after screenshots in the description?

**Comment by bicknellr** [comment]
@cdata added screenshots

File: demo/paper-tabs-demo-styles.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <template>
     <style>
       :root {
-        --paper-toolbar-background: #00bcd4;
+        --paper-toolbar-background: var(--paper-light-blue-800);
```
Comment (cdata) [inline demo/paper-tabs-demo-styles.html]: :+1:

File: demo/paper-tabs-demo-styles.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
       }
 
       paper-tabs, paper-toolbar {
-        background-color: #00bcd4;
+        background-color: var(--paper-light-blue-800);
```
Comment (cdata) [inline demo/paper-tabs-demo-styles.html]: :beers:

**Comment by cdata** [comment]
LGTM :+1:

**Comment by alice** [comment]
I measure the contrast ratio of the semi-transparent white on darker blue here as 3.3: http://leaverou.github.io/contrast-ratio/#%23BEDDEE-on-%23007ABE

**Comment by bicknellr** [comment]
I think those colors are adjusted for your monitor's color profile; [#0277bd](https://github.com/PolymerElements/paper-styles/blob/d253ec2c0370f32f5ced2cc5e53c04e6adf6fdd4/color.html#L144) with #fff@0.8 blended on top gives about 3.6.

**Comment by alice** [comment]
3.6 is still pretty low. We should be shooting for 4.5.

**Comment by bicknellr** [comment]
Ok: #104. http://leaverou.github.io/contrast-ratio/#%23cad6ea-on-%230d47a1

**Pull request merged**